spec/models/work_spec.rb

```diff
@@ -185,8 +185,18 @@
     end
 
     it "should find works imported with irrelevant query parameters" do
-      work = create(:work, imported_from_url: 'http://lj-site.com/thing1?style=mine')
-      expect(Work.find_by_url('http://lj-site.com/thing1?style=other')).to eq(work)
+      url = "http://lj-site.com/thing1?style=mine"
```

Comment:
I'm confused - why have you changed this test? the whole point of it is to check that the `find_by_url` method sees "lj-site.com/thing1?**style=mine**" as the same as "lj-site.com/thing1?**style=other**". Can you change it back to how it was?